easybuild/easyblocks/o/openblas.py

```diff
@@ -94,3 +168,132 @@ def sanity_check_step(self):
             'dirs': [],
         }
         super(EB_OpenBLAS, self).sanity_check_step(custom_paths=custom_paths)
+
+    def _optarch_for_compiler(self, optarch):
+        """
+        Extracts the appropriate optarch for the compiler currently being used.
+        If it is not compiler-specific it is returned as-is.
+        If no optarch is found, False is returned.
+        :param optarch: A complete optarch statement.
+        https://easybuild.readthedocs.io/en/latest/Controlling_compiler_optimization_flags.html
+        """
+        if optarch is False:
```

Comment:
For me (on POWER) optarch is `None`, so maybe `if not optarch:`? And I'd return `None` too as that seems to better fit the meaning

Reply:
Yep, certainly.